visualizations/frontend/filters/TRangeSelector.vue

```diff
@@ -15,7 +15,11 @@
           class="flex items-center gap-1 font-normal"
         >
           : {{ barMinValue }}-{{ barMaxValue }}
-          <close-icon :size="12" @click="removeFilter" />
+          <reload-icon
```

Comment:
These changes were already pushed in previous [PR](https://github.com/topcoat-data/topcoat-public/commit/eb46c91648a3d95e9a8ae8a0abb33f971adfe33f)

Reply:
Only TFiltersDropdown changes required.